css/css-variables/variable-css-wide-keywords.html

```diff
@@ -0,0 +1,330 @@
+<!DOCTYPE html>
+<head>
+<title>CSS Custom Properties: Using CSS-wide keywords</title>
+<link rel="help" href="https://drafts.csswg.org/css-variables/#ref-for-css-wide-keywords%E2%91%A0">
```

Comment:
I worry a little about the ① which is how Bikeshed de-dupes ids.

Reply:
Yes, please don't use those `#ref-for` IDs, they're auto-generated and not remotely stable, and created just so the index can link to those elements. It would be more reasonable to use a [text fragment](https://developer.mozilla.org/en-US/docs/Web/Text_fragments); then at least it wouldn't break when someone added or removed a reference above that point in the spec.

But really, if you need an anchor to that paragraph, we can just add an id to the paragraph. Please do that instead.

Reply:
@tabatkins I'll be happy to change this. I did not quickly find a way to add an `id` to a paragraph in bikeshed, what could be the best way to do this in this case?

Here is this place in the raw spec: https://github.com/w3c/csswg-drafts/blob/main/css-variables-1/Overview.bs#L219

I wonder if there is a way to make the ref link in this case to be stable instead (provide a hard-coded id to it?)

Or maybe a more simple fix in this case would be for me to just link to the section containing this paragraph, in this case, https://drafts.csswg.org/css-variables/#defining-variables ?

Reply:
Maybe the simple answer is:
* link to the section
* quote the spec sentence in a `<meta name=assert>` [as documented](https://web-platform-tests.org/writing-tests/css-metadata.html#test-assertions)

Reply:
That said, if you want to add an `id` to the paragraph in the spec I think you can just start it with `<p id="my-id">`.

Reply:
Yeah, I propose linking to a section. But if you really do want to link to a paragraph, the answer is "add a `<p>` tag" - just because we use Markdown paragraphs for nearly everything doesn't mean HTML paragraphs aren't allowed. ^_^

Reply:
Done: changed the link to point to the section, and added a `<meta name="assert">`